src/models/model_base.py

```diff
@@ -0,0 +1,22 @@
+from src import db
+
+
+class ModelBase(db.Model):
+    __abstract__ = True
+
+    def save_to_db(self):
+        db.session.add(self)
+        db.session.commit()
+
+    def delete_from_db(self):
+        db.session.delete(self)
+        db.session.commit()
+
+    # filter_by(...).update(...) is not working in inheritance
```

Comment:
# filter_by(...).update(...) is not working in inheritance
I think I might know the problem with this.
https://hackersandslackers.com/database-queries-sqlalchemy-orm/#:~:text=Unlike%20filter()%20%2C%20filter_by(),match%20the%20arguments%20we%20pass
Unlike filter(), filter_by() accepts keyword arguments (note the difference in syntax here: filter() checks a conditional against a column object whereas filter_by() finds columns which match the arguments we pass). filter_by() can only search for exact values and serves as a kind of shorthand for simple filtering queries.